package.json

```diff
@@ -22,6 +22,7 @@
 		"fix": "run-p --continue-on-error fix:*",
 		"fix:eslint-docs": "eslint-doc-generator",
 		"fix:js": "npm run lint:js -- --fix",
+		"fix:snapshots": "ava --update-snapshots",
```

Comment:
I'm also including the snapshots in the `fix` command, which I thought was enough initially but it did not update the snapshots